### ISSUE TITLE
How train the network

### ISSUE THREAD

**Issue by Mr-Bang** [albert100121/MLVR-Pytorch#1]
Hello albert, thank you very much for your effort! I want to know how to train this network,(where to set the learning rate and etc.)
Thanks again!
